<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>500</width>
    <height>350</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>500</width>
    <height>350</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>500</width>
    <height>350</height>
   </size>
  </property>
  <property name="font">
   <font>
    <family>Microsoft YaHei UI</family>
   </font>
  </property>
  <property name="windowTitle">
   <string>Facial Recognization</string>
  </property>
  <property name="windowIcon">
   <iconset resource="images.qrc">
    <normaloff>:/images/fire.png</normaloff>:/images/fire.png</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="Line" name="HeaderLine">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>24</y>
      <width>500</width>
      <height>8</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>500</width>
      <height>8</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>500</width>
      <height>8</height>
     </size>
    </property>
    <property name="lineWidth">
     <number>1</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QLabel" name="HeaderText">
    <property name="geometry">
     <rect>
      <x>193</x>
      <y>0</y>
      <width>114</width>
      <height>26</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Microsoft YaHei UI</family>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color:#ffffff</string>
    </property>
    <property name="text">
     <string>Facial Recognition</string>
    </property>
   </widget>
   <widget class="QLabel" name="ImgLabel">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>40</y>
      <width>200</width>
      <height>200</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>200</width>
      <height>200</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>200</width>
      <height>200</height>
     </size>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="ButtonGetResult">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>210</y>
      <width>80</width>
      <height>30</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>80</width>
      <height>30</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>80</width>
      <height>30</height>
     </size>
    </property>
    <property name="font">
     <font>
      <family>Microsoft YaHei UI</family>
      <pointsize>9</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);
background-color: rgb(85, 85, 127);</string>
    </property>
    <property name="text">
     <string>Get Results</string>
    </property>
   </widget>
   <widget class="Line" name="leftline">
    <property name="geometry">
     <rect>
      <x>376</x>
      <y>27</y>
      <width>8</width>
      <height>323</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>8</width>
      <height>323</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>8</width>
      <height>323</height>
     </size>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="Line" name="rightline">
    <property name="geometry">
     <rect>
      <x>116</x>
      <y>27</y>
      <width>8</width>
      <height>323</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>8</width>
      <height>323</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>8</width>
      <height>323</height>
     </size>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="QPushButton" name="ButtonChooseFiles">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>290</y>
      <width>80</width>
      <height>30</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>80</width>
      <height>30</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>80</width>
      <height>30</height>
     </size>
    </property>
    <property name="font">
     <font>
      <family>Microsoft YaHei UI</family>
      <pointsize>9</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(85, 85, 127);
color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>Image-Files</string>
    </property>
   </widget>
   <widget class="QListView" name="SpaceBottom">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>328</y>
      <width>500</width>
      <height>24</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>500</width>
      <height>24</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>500</width>
      <height>24</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">background:black;</string>
    </property>
   </widget>
   <widget class="QPushButton" name="ButtonQuitBottom">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>290</y>
      <width>80</width>
      <height>30</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>80</width>
      <height>30</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>80</width>
      <height>30</height>
     </size>
    </property>
    <property name="font">
     <font>
      <family>Microsoft YaHei UI</family>
      <pointsize>9</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);
background-color: rgb(85, 85, 127);</string>
    </property>
    <property name="text">
     <string>Exit</string>
    </property>
   </widget>
   <widget class="QLabel" name="TextBottomleft">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>328</y>
      <width>130</width>
      <height>24</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Microsoft YaHei UI</family>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color:#ffffff</string>
    </property>
    <property name="text">
     <string>Â©2021 Zsbyqx-THUEE</string>
    </property>
   </widget>
   <widget class="QListView" name="HeaderIcon">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>26</width>
      <height>26</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>26</width>
      <height>26</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>26</width>
      <height>26</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/images/fire.png);</string>
    </property>
   </widget>
   <widget class="QListView" name="HeaderSpace">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>500</width>
      <height>28</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>500</width>
      <height>28</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>500</width>
      <height>28</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">background:black;</string>
    </property>
   </widget>
   <widget class="QPushButton" name="ButtonMiniTop">
    <property name="geometry">
     <rect>
      <x>435</x>
      <y>1</y>
      <width>26</width>
      <height>25</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>26</width>
      <height>25</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>26</width>
      <height>25</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(0, 0, 0);
border-image: url(:/images/heavy_minus_sign.png);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="ButtonQuitTop">
    <property name="geometry">
     <rect>
      <x>474</x>
      <y>1</y>
      <width>26</width>
      <height>25</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>26</width>
      <height>25</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>26</width>
      <height>25</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(0, 0, 0);
border-image: url(:/images/x.png);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="labelImgName">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>240</y>
      <width>200</width>
      <height>24</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>200</width>
      <height>24</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>200</width>
      <height>24</height>
     </size>
    </property>
    <property name="font">
     <font>
      <family>Microsoft YaHei</family>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);
background-color: rgba(170, 170, 255, 150);</string>
    </property>
    <property name="text">
     <string>Name:</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelExpGender">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>264</y>
      <width>200</width>
      <height>24</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>200</width>
      <height>24</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>200</width>
      <height>24</height>
     </size>
    </property>
    <property name="font">
     <font>
      <family>Microsoft YaHei</family>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);
background-color: rgba(170, 170, 255, 150);</string>
    </property>
    <property name="text">
     <string>Expected Gender:</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelThouGender">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>288</y>
      <width>200</width>
      <height>24</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>200</width>
      <height>24</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>200</width>
      <height>24</height>
     </size>
    </property>
    <property name="font">
     <font>
      <family>Microsoft YaHei</family>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);
background-color: rgba(170, 170, 255, 150);</string>
    </property>
    <property name="text">
     <string>Predicted Gender:</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBoxQuickChoose">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>160</y>
      <width>100</width>
      <height>80</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>100</width>
      <height>80</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>100</width>
      <height>80</height>
     </size>
    </property>
    <property name="font">
     <font>
      <family>Microsoft YaHei</family>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QGroupBox{
color: rgb(255, 255, 255);
}</string>
    </property>
    <property name="title">
     <string>Quick-Choose</string>
    </property>
    <widget class="QComboBox" name="comboBoxGender">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>80</width>
       <height>20</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>80</width>
       <height>20</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>80</width>
       <height>20</height>
      </size>
     </property>
     <property name="font">
      <font>
       <family>Microsoft YaHei</family>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(255, 255, 255);
background-color: rgb(85, 85, 127);</string>
     </property>
     <item>
      <property name="text">
       <string>Female</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Male</string>
      </property>
     </item>
    </widget>
    <widget class="QComboBox" name="comboBoxPath">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>50</y>
       <width>80</width>
       <height>20</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>80</width>
       <height>20</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>80</width>
       <height>20</height>
      </size>
     </property>
     <property name="font">
      <font>
       <family>Microsoft YaHei</family>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(255, 255, 255);
background-color: rgb(85, 85, 127);</string>
     </property>
    </widget>
   </widget>
   <widget class="QLabel" name="TextBottomright">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>326</y>
      <width>130</width>
      <height>24</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Microsoft YaHei UI</family>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color:#ffffff</string>
    </property>
    <property name="text">
     <string>Python  </string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QListView" name="listViewBgImg">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>27</y>
      <width>500</width>
      <height>302</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/images/milky.jpg);</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelImpleRight">
    <property name="geometry">
     <rect>
      <x>307</x>
      <y>0</y>
      <width>128</width>
      <height>26</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelImpleLeft">
    <property name="geometry">
     <rect>
      <x>26</x>
      <y>0</y>
      <width>167</width>
      <height>26</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="Line" name="line_h1">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>262</y>
      <width>200</width>
      <height>4</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="Line" name="line_h2">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>286</y>
      <width>200</width>
      <height>4</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="Line" name="line_h3">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>238</y>
      <width>200</width>
      <height>4</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="Line" name="line_h4">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>310</y>
      <width>200</width>
      <height>4</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="Line" name="line">
    <property name="geometry">
     <rect>
      <x>349</x>
      <y>240</y>
      <width>2</width>
      <height>72</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="Line" name="line_2">
    <property name="geometry">
     <rect>
      <x>149</x>
      <y>240</y>
      <width>2</width>
      <height>72</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <zorder>listViewBgImg</zorder>
   <zorder>HeaderSpace</zorder>
   <zorder>HeaderLine</zorder>
   <zorder>HeaderText</zorder>
   <zorder>ImgLabel</zorder>
   <zorder>ButtonGetResult</zorder>
   <zorder>leftline</zorder>
   <zorder>rightline</zorder>
   <zorder>ButtonChooseFiles</zorder>
   <zorder>SpaceBottom</zorder>
   <zorder>ButtonQuitBottom</zorder>
   <zorder>TextBottomleft</zorder>
   <zorder>HeaderIcon</zorder>
   <zorder>ButtonMiniTop</zorder>
   <zorder>ButtonQuitTop</zorder>
   <zorder>labelImgName</zorder>
   <zorder>labelExpGender</zorder>
   <zorder>labelThouGender</zorder>
   <zorder>groupBoxQuickChoose</zorder>
   <zorder>TextBottomright</zorder>
   <zorder>labelImpleRight</zorder>
   <zorder>labelImpleLeft</zorder>
   <zorder>line_h1</zorder>
   <zorder>line_h2</zorder>
   <zorder>line_h3</zorder>
   <zorder>line_h4</zorder>
   <zorder>line</zorder>
   <zorder>line_2</zorder>
  </widget>
 </widget>
 <resources>
  <include location="images.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>ButtonQuitBottom</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>414</x>
     <y>305</y>
    </hint>
    <hint type="destinationlabel">
     <x>363</x>
     <y>305</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>ButtonQuitTop</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>484</x>
     <y>18</y>
    </hint>
    <hint type="destinationlabel">
     <x>485</x>
     <y>58</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>ButtonMiniTop</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>showMinimized()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>406</x>
     <y>13</y>
    </hint>
    <hint type="destinationlabel">
     <x>418</x>
     <y>-3</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
